boot.js

```diff
@@ -50,3 +52,11 @@ artsyXapp.on('error', (error) => {
   console.warn(error)
   process.exit(1)
 })
+
+// TODO: Possibly a terrible hack to not share `req.user` between both.
+const doNotShareUserHack = (app) => {
+  app.use((req, rest, next) => {
+    req.user = null
+    next()
+  })
+}
```

Comment:
Ah right this comment can probably be removed since it's pretty old and we actually benefit from not sharing req.user because it keeps our API stateless.